enigma-swing/src/main/java/cuchaz/enigma/gui/config/UiConfig.java

```diff
@@ -6,10 +6,66 @@
 
 import cuchaz.enigma.config.ConfigContainer;
 import cuchaz.enigma.config.ConfigSection;
+import cuchaz.enigma.gui.panels.right.RightPanel;
 import cuchaz.enigma.gui.util.ScaleUtil;
 import cuchaz.enigma.utils.I18n;
 
 public final class UiConfig {
+	// sections
+	public static final String MAIN_WINDOW = "Main Window";
```

Comment:
Is there any reason for these fields to be public?

Reply:
the entire reason to have them public is the main window constant being used in `Gui`
you know me and my consistency